packages/verbum-block-editor/package.json

```diff
@@ -23,15 +23,17 @@
 	"bugs": "https://github.com/Automattic/wp-calypso/issues",
 	"types": "dist/types",
 	"scripts": {
-		"clean": "rm -rf dist",
-		"build": "NODE_ENV=production yarn dev",
+		"clean": "",
```

Comment:
we would need to revert this right?

Reply:
Not now at least. If we keep this, `yarn dev` will delete the `dist` folder and the translations with it, and your sandbox won't have them. Once we deploy, we can put this back since `yarn dev --sync` won't delete files on remote if they don't exist on your site.